README.md

```diff
@@ -36,6 +36,7 @@ The top level of the configuration file looks like this:
   ...
   "snaps": [
     {
+      "nameIdentifier": "myUniqueSnapName",
```

Comment:
Proposal: maybe we can name this parameter `__snapName`, since it is more of an “internal” parameter like the others are.